pkgs/development/python-modules/ifconfig-parser/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "ifconfig-parser";
+  version = "0.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12n52zqwwc2k8si74qlgx2c33mc9glnkx23gn55k2zqdv892ylbc";
+  };
```

Comment:
pypi sdist doesnt include tests, please pull from github and run unit tests. Important when bumping dependencies.

```
running egg_info
writing ifconfig_parser.egg-info/PKG-INFO
writing dependency_links to ifconfig_parser.egg-info/dependency_links.txt
writing top-level names to ifconfig_parser.egg-info/top_level.txt
reading manifest file 'ifconfig_parser.egg-info/SOURCES.txt'
writing manifest file 'ifconfig_parser.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Finished executing setuptoolsCheckPhase
derivation '/nix/store/z2p3y5y32msdd1w82kgncrdnx1cdc3gr-python3.7-ifconfig-parser-0.0.5.drv'
```

Reply:
I tried that but it didn't run the tests either eventhough they're now in the src.

Jc has tests for this package's functionality though.

Reply:
there are tests here https://github.com/KnightWhoSayNi/ifconfig-parser/tree/master/ifconfigparser/tests

Reply:
Yeah but they don't get run by the checking phase. I'm not familiar enough with python packaging yet to make them go manually, how would one do that?

Reply:
you can look at their CI to see how they call the tests https://github.com/KnightWhoSayNi/ifconfig-parser/blob/master/.travis.yml